### DISCUSSION TITLE
How to turn of Hero 10 after Nightlapse completes

### DISCUSSION THREAD

**Discussion by Cameratrapsue** [gopro/labs#872] [Q&A] [answered]
I have a Hero 10 running the latest firmware and labs.
I'm trying to have the camera come on at a predetermined time such as 9:15pm then start running the Nightlapse settings I have already set in camera (camera is set to Nightlapse as default). It should run the Nightlapse all night until say, 5:15am at that time it should stop taking night photos then wait maybe a minute then turn off the camera completely.

I can get the camera to turn on at 9:15pm and turn off at 05:15am but it will not turn off the camera a minute or so later.

The code I've been trying is !21:15N!S!05:15E!05:170 

I'm doing something wrong but can't figure out what, I've played around with Blockly but am such a newbie that I am not getting very far their either though surprisingly I have had some success with camera on and off at needed times.

Do you know a QR code that could do the above? and if so any way I could see it in Blockly to help me learn what I'm doing wrong.

Thanks!

**Comment by dnewman-gpsw** [comment, marked as answer]
O not 0  (O for Off, on 0 for zero)

Try ```!21:15N!S!05:15E!05:17O```

**Comment by Cameratrapsue** [comment]
Thank you! that works great. I tested it using daytime hours and everything worked fine. I appreciate your fast reply so I can try this out tonight.
